test/ngMock/angular-mocksSpec.js

```diff
@@ -949,7 +948,7 @@ describe('ngMock', function() {
           if (!error.stack) {
             try {
               throw error;
-            } catch (e) {}
+            } catch (e) { /** do nothing **/}
```

Comment:
Any comment suffices.

On Mon, Aug 1, 2016 at 2:46 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In test/ngMock/angular-mocksSpec.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72971346>:
>
> > @@ -949,7 +948,7 @@ describe('ngMock', function() {
> >            if (!error.stack) {
> >              try {
> >                throw error;
> > -            } catch (e) {}
> > +            } catch (e) { /** do nothing **/}
>
> No /* empty */ here?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73#r72971346>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnmR8E7lsK2CgBZleNvCbAlM6nU9Iks5qbeqngaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski